examples/let-clause/let_clause.md

```diff
@@ -1,7 +1,19 @@
 # Let clause
 
-Query expressions can have `let` clauses. They can be anywhere between `from` and `select` clauses. Multiple `where` clauses are also allowed. The semantics are similar to `XQuery FLWOR`.
+Let clauses can be used to define temporary variables inside a query expression . It can occur multiple times anywhere between from and select clause. The semantics are similar to XQuery FLWOR.
```

Comment:
```suggestion
`let` clause can be used to define temporary variables inside a query expression. It can occur multiple times anywhere between the `from` and `select` clauses. The semantics are similar to XQuery FLWOR.
```